src/public/meeting.ts

```diff
@@ -325,4 +349,22 @@ export namespace meeting {
     ensureInitialized(FrameContexts.sidePanel);
     sendMessageToParent('meeting.getAppContentStageSharingState', callback);
   }
+
+  /**
+   * Registers a handler for changes to speaking array.
+   * @param handler The handler to invoke when the array of participants speaking changes. A value
+   * of 0 (no participants speaking) or 1 (one or more participants are speaking) is provided.
+   * @param includeParticipants is a flag to opt-in to receive participant ids of who is speaking.
+   * Default value is false.
+   */
+  export function registerDetectSpeakingStateChangedHandler(
+    handler: (isSpeakingDetected: ISpeakingState) => void,
+    includeParticipants?: boolean,
```

Comment:
Why would the caller specify `false` here instead of just ignoring the information in the callback they provide? Does it make it cheaper on the Teams' side not to send participants info?

Reply:
No, this was more to do with permissions. Getting the participants info would require an additional permission in the app manifest.

Reply:
hmm, somehow that doesn't sit well with me. Should there be two separate APIs instead?

Reply:
Hi @MarcinJuraszek thank you for the feedback! I updated it to remove that part of including participants. If needed in the future, it can be two separate APIs. For now, we will only expose if anyone is speaking, or no one is speaking.